contributing/Dockerfile

```diff
@@ -1,27 +1,25 @@
-FROM ubuntu:18.04
+ARG NODE_VERSION=16.15.1
+FROM node:${NODE_VERSION}-slim
```

Comment:
Any advantage of doing it like this? 

I find this a bit harder to understand when we're starting with a Node image and extending it rather than starting with a distro and adding Node.

Questions that pop in my head:
- What distro am I working with? Is it Alpine, Ubuntu, Centos, etc.

Reply:
Awesome, thanks so much for the context @wheinze. I had no idea Debian was Ubuntu 😂

Yes, we will be getting rid of the Node.js requirement in the coming weeks. When that happens, would we just need to update the it to `FROM debian-slim` or something?

Reply:
> Yes, we will be getting rid of the Node.js requirement soon. When that happens, would we just need to update the it to `FROM debian-slim` or something?

When we reach this state, I would recommend to switch over to [`golang:1.18.3`](https://hub.docker.com/_/golang), which is also based on Debian or preferable even `golang:1.18.3-alpine` which in the end saves us time as we don't need to setup Go anymore.

> An alpine image would be :chefkiss:

Would volunteer to work on that ;)

Reply:
Looks good! Sounds like there's a good migration path then.

> Would volunteer to work on that ;)
Appreciate it! You're quickly becoming my Devops guru 🧘‍♂️

Do you think there is an advantage to having multiple Dockerfiles in contributing? I'm tempted to just have one that's the most optimized we can make it.